Add Comment to readme.md (#18)
bpfkorea/boa-sdk-ts: README.md

# boa-sdk-ts
SDK to interface with the BOSAGORA blockchain

> Boa-sdk-ts is a TypeScript/JavaScript library for communicating with the Stoa API server.

Reviewer: ```suggestion
boa-sdk-ts is a TypeScript/JavaScript library for communicating with the Stoa API server.
```
The name of the package is in lower case, so I'd like to write it in lower case

Author: Thank you. Applied the suggestion.

> It also has a hashing function and a pre-image validation.

Reviewer 2: ```suggestion
It also has utility functions such as hashing and pre-image validation.
```

Author: Thank you. Applied the suggestion.

## Install

> ```npm install --save boa-sdk-ts```

Reviewer: ```suggestion
```bash
npm install --save boa-sdk-ts
```

```

> ## Import the your TypeScript

Reviewer 2: ```suggestion
## Import the library
```

Author: Thank you. Applied the suggestion.

```import * as BoaSdk from "boa-sdk-ts";```

## Usage
```TypeScript
// Set URL
let uri = URI("http://localhost").port(port);
// Create BOA Client
let boa_client = new BoaSdk.BOAClient(uri.toString());

// Query
boa_client.getValidator("GA3DMXTREDC4AIUTHRFIXCKWKF7BDIXRWM2KLV74OPK2OKDM2VJ235GN", 10)
.then((validators: Array<BoaSdk.Validator>) =>
{
    // On Success
})
.catch(err =>
{
    // On Error
});
```

## Testing

> ```

Reviewer: ```suggestion
```bash
```

# boa-sdk-ts
SDK to interface with the BOSAGORA blockchain
boa-sdk-ts is a TypeScript/JavaScript library for communicating with the Stoa API server.
It also has utility functions such as hashing and pre-image validation.

## Install
```bash
$ npm install --save boa-sdk-ts
```

## Import the your library
```import * as BoaSdk from "boa-sdk-ts";```

## Usage
```TypeScript

> // Set URL
> let uri = URI("http://localhost").port(port);
> // Create BOA Client
> let boa_client = new BoaSdk.BOAClient(uri.toString());

Reviewer: ```suggestion
// Create BOA Client
let boa_client = new BoaSdk.BOAClient("http://localhost:3836");
```

Author: Applied the suggestion.
